api/BUILD

```diff
@@ -147,6 +148,7 @@ proto_library(
         "//envoy/extensions/filters/network/tcp_proxy/v3alpha:pkg",
         "//envoy/extensions/filters/network/thrift_proxy/v3alpha:pkg",
         "//envoy/extensions/filters/network/zookeeper_proxy/v3alpha:pkg",
+        "//envoy/extensions/retry/omit_host_metadata/v3alpha:pkg",
```

Comment:
This should reflect the source/extensions organization, so the package should be under `retry/host/omit_host_metadata`.

Reply:
Done